src/lib/storage/vector_compression/simd_bp128/simd_bp128_iterator.hpp

```diff
@@ -18,77 +19,32 @@ class SimdBp128Iterator : public BaseCompressedVectorIterator<SimdBp128Iterator>
   using Packing = SimdBp128Packing;
 
  public:
-  SimdBp128Iterator(const pmr_vector<uint128_t>* data, const size_t size, const size_t absolute_index = 0u);
-
+  // The SimdBp128Iterator simply wraps a SimdBp128Decompressor (which implements the logic to cache and extract
+  // blocks/meta blocks) to allow STL algorithms (e.g., std::lower_bound) to be used on a compressed vector. However,
+  // please be aware of #1531. Other vector compression containers such as FSBA use std::vector for the underlying data
```

Comment:
“Such as”? :D

Reply:
```
// The SimdBp128Iterator simply wraps a SimdBp128Decompressor (which implements the logic to cache and extract
  // blocks/meta blocks) to allow STL algorithms (e.g., std::lower_bound) to be used on a compressed vector. However,
  // please be aware of #1531. The fixed-size-byte-aligned vector compression uses std::vector for the underlying data
  // and dooes thus not require an additional iterator.
```

Reply:
Ok?

Reply:
typo dooes, otherwise ok

Reply:
You haven't changed this yet, have you?

Reply:
Did change, but haven’t pushed. I am not on a PC right now.